crates/kubelet/src/store/mod.rs

```diff
@@ -1,14 +1,53 @@
 //! `store` contains logic around fetching and storing modules.
 pub mod oci;
 
+use oci_distribution::client::ImageData;
 use std::collections::HashMap;
 use std::convert::TryFrom;
+use std::sync::Arc;
+use tokio::sync::Mutex;
+use tokio::sync::RwLock;
 
 use async_trait::async_trait;
 use log::debug;
 use oci_distribution::Reference;
 
 use crate::pod::Pod;
+use crate::store::oci::Client;
+
+/// Specifies how the store should check for module updates
+#[derive(PartialEq, Debug)]
+pub enum ModulePullPolicy {
```

Comment:
Since this can also be used for non-module projects without any changes, should we call this a `PullPolicy` instead?

I imagine you could use this with OCI images as well, for example.